test/MarketMaker.js

```diff
@@ -236,19 +223,15 @@ contract('marketMaker', function(accounts) {
     state.ethBalances[customerAddressIndex] = web3.eth.getBalance(customer);
     state.balances[customerAddressIndex] = await token.balanceOf(customer);
 
-    const startBlock = web3.eth.blockNumber + 10;
-    const blocksPerPeriod = 15;
+    const startTimestamp = parseInt(await mm.startTimestamp.call());
+    const secondsPerPeriod = duration.days(30);
 
     const foundationWallet = accounts[9];
 
-    mm = await LifMarketMaker.new(token.address, startBlock, blocksPerPeriod, periods,
-      foundationWallet, {from: accounts[0]});
+    await mm.transferOwnership(foundationWallet, {from: accounts[0]});
```

Comment:
Is this needed? Seems the crowdsale is taking care of doing this now

Reply:
not needed, removed :)